### PR TITLE
feat: add sm and md size attributes

### PR DESCRIPTION
**Pull request by ecommerceUXdesign**
# Alaska Airlines Pull Request

**Fixes:** #2 #6 

## Summary:

Updates dialog visibility support to include a `sm` and `md` size setting. Default is `lg`. 

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by blackfalcon** [comment]
PR is ready for review. Need an assist with tests.

File: src/auro-dialog.js [hunk 1]
```diff
@@ -85,20 +85,59 @@ class AuroDialog extends LitElement {
...
+        this.dialog.classList.add('dialog--hidden');
+      }, 500);
+    }
+  }
+
+  /**
+   * @private function for setting aria state
+   * @returns {String} - Returns state string value
+   */
+  aria() {
```
Comment (geoffrich) [inline src/auro-dialog.js:125]: We should remove aria-hidden altogether when the modal is open. `aria-hidden="false"` [currently behaves inconsistently across browsers.](https://www.w3.org/TR/wai-aria-1.1/#aria-hidden)

Comment (geoffrich) [reply src/auro-dialog.js:125]: Testing with Firefox/NVDA on the demo site, I was unable to read out the contents of an open modal.

File: src/style.scss [hunk 2]
```diff
@@ -129,6 +129,10 @@ $auro-spacing-options: none;
   z-index: -1;
   overflow: auto;
 
+  &--hidden {
```
Comment (geoffrich) [inline src/style.scss:130]: This class doesn't appear to be used anywhere, which is why I can still tab into a closed modal.

**Review by geoffrich — CHANGES_REQUESTED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**